invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html

```diff
@@ -35,24 +35,28 @@
 {% macro show_creatibutors(creatibutors, show_affiliations=False) %}
   {% for creatibutor in creatibutors %}
   <dd class="creatibutor-wrap">
-    {{ creatibutor_icon(creatibutor) }}
-    <a class="ui tooltip-popup text-muted creatibutor-link"
+    <a class="ui tooltip-popup creatibutor-link"
       {% if show_affiliations and creatibutor.affiliations %}
         data-content="{{ creatibutor.affiliations|join('; ', attribute='1') }}"
       {% endif %}
       href="{{url_for('invenio_search_ui.search', q='metadata.creators.person_or_org.name:"' + creatibutor.person_or_org.name + '"')}}"
     >
 
       <span class="creatibutor-name">
-        {{ creatibutor.person_or_org.name }}
+        {{- creatibutor.person_or_org.name -}}
       </span>
-
-      <sup class="font-tiny text-muted">
-        {% for affiliation in creatibutor.affiliations %}
-          {{ affiliation[0] }}{{ ", " if not loop.last }}
-        {% endfor %}
+      {%- if creatibutor.affiliations -%}
+      <sup class="font-tiny">
+        {%- for affiliation in creatibutor.affiliations -%}
+          {{- affiliation[0] -}}{{- ", " if not loop.last -}}
+        {%- endfor -%}
       </sup>
+      {%- endif -%}
     </a>
+    {{- creatibutor_icon(creatibutor) -}}
+    {%- if not loop.last -%}
+       <span class="separator">;</span>
```

Comment:
I guess the `;` is making a comeback @jennur ;) . Jenny and I were initially trying to tone down the punctuation and make use of whitespace + the contributor icon to separate each contributor. I think I still lean that way, but we can discuss.

Reply:
Oh, I see, I hadn't realized. One factor is that they won't all have ORCID or group icons. Yes let's discuss, and also discuss the best way to make sure it's straightforward to theme.

Reply:
I too suggest to rather add some extra space to separate the creatibutors. Even if we are to keep the `;` I think some extra white space is needed between the creatibutors.

Reply:
Sounds good; it's now bumped from .5rem to 1rem